### PR TITLE
QuickPi - Changed category 'Action' to 'Actuators' which is more relevant

### PR DESCRIPTION
**Pull request by NicolasWent**
The blockly category "action" has been renamed to "actuator", which is a more relevant name for these components.

I needed to add the "actuator" category for all languages, but since now, actuator only exist in QuickPi, so only QuickPi is normally impacted by these changes.

For the translations, the file and line to modify for translation is referenced inside of the Google Sheet.

### CODE REVIEW

File: pemFioi/quickAlgo/i18n.js [hunk 1]
```diff
@@ -6,7 +6,8 @@
 var quickAlgoLanguageStrings = {
    fr: {
       categories: {
-         actions: "Actions",
+         actions: "Action",
```
Comment (mblockelet) [inline pemFioi/quickAlgo/i18n.js]: When you commit, check that you didn't accidentally modify lines you didn't intend to.

**Review by mblockelet — CHANGES_REQUESTED** [review]
Apart from that line which shouldn't have been modified, everything is good for me.

File: pemFioi/quickAlgo/i18n.js (hunk 1 above)
Comment (NicolasWent) [reply pemFioi/quickAlgo/i18n.js]: Ok, didn't saw that thank you.

The line has been modified in the last commit.

**Review by mblockelet — APPROVED** [review]

**Review by smadbe — APPROVED** [review]
For such a simple word, I think it would worth translating it in German, Spanish, ... using google translate. (to prevent missing translation later)

**Review by smadbe — APPROVED** [review]
great

**Review by mblockelet — APPROVED** [review]

**Pull request merged**